solutions/usaco-1037.mdx

```diff
@@ -2,19 +2,98 @@
 id: usaco-1037
 source: USACO Bronze 2020 US Open
 title: Cowntact Tracing
-author: Brad Ma
+author: Brad Ma, Ryan Chou
 ---
 
 [Official Analysis (C++)](http://usaco.org/current/data/sol_tracing_bronze_open20.html)
 
+By simulating handshakes for all values of $K$, we can find the minimum, maximum, and number of possible values of $K$ which are vaild.
 ## Implementation
 
 **Time Complexity:** $\mathcal{O}(N \cdot T^2)$
 
 <LanguageSection>
 
+<PySection>
+
+```py
+import sys
+
+sys.stdin = open('tracing.in', 'r')
+sys.stdout = open('tracing.out', 'w')
+
+n, t = map(int, input().split())
+
+infected = [int(i) for i in input().strip()]
+shakes = []
+
+for _ in range(t):
+	shakes.append(list(map(int, input().split())))
+
+# sorting a matrix automatically sorts lists the first element.
```

Comment:
? did you mean sorts by the first element or smth?

Reply:
yeah

Reply:
fixed it.